spec/models/parent_spec.rb

```diff
@@ -17,7 +17,7 @@
 #  updated_at           :datetime         not null
 #
 
-describe Parent do
+describe Parent, type: :model do
```

Comment:
Isn't the type inferred from the file location?

Reply:
Oh, well, I've learnt something new today!